modules/installation-configuration-parameters.adoc

```diff
@@ -338,6 +338,11 @@ networking:
   - fd00:172:16::/112
 ----
 
+[NOTE]
+====
+Globalnet is not supported with Red Hat OpenShift Data Foundation disaster recovery solutions. Ensure that you use a non-overlapping range of private IP addresses for the cluster and service networks in each cluster, for regional disaster recovery scenarios.
```

Comment:
- Is "Globalnet" in this note the company "[GlobalNet](https://www.gblnet.net/en)"? if so, capitalization should be updated.
- "Red Hat OpenShift Data Foundation" has an [attribute](https://github.com/openshift/openshift-docs/blob/main/_attributes/common-attributes.adoc), `rh-storage-first`
- Also, a couple of small style tweaks:

```suggestion
Globalnet is not supported with {rh-storage-first} disaster recovery solutions. For regional disaster recovery scenarios, ensure that you use a nonoverlapping range of private IP addresses for the cluster and service networks in each cluster.
```

Reply:
- This Globalnet is not the company GlobalNet.
- Addressed
- Addressed